Commandant.xcodeproj/project.pbxproj

```diff
@@ -430,6 +433,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "org.carthage.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 3.0;
```

Comment:
Setting the value at the project level, not at each target level, would be fine as same as #68.